experimental/adaptive-dialog/csharp_dotnetcore/todo-bot/NuGet.Config

```diff
@@ -2,6 +2,6 @@
 <configuration>
   <packageSources>
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
-    <add key="BotBuilder.myget.org" value="https://botbuilder.myget.org/F/botbuilder-declarative/api/v3/index.json" protocolVersion="3" />
+    <add key="BotBuilder.myget.org" value="https://botbuilder.myget.org/F/botbuilder-v4-dotnet-daily/api/v3/index.json" protocolVersion="3" />
```

Comment:
Adaptive are on top of 4.6 (4.6.0-preview2). Should we use regular packages and not myget and/or daily builds?

Reply:
I'll make a separate PR to update the csproj files once we have final packages on myget.